src/component/handlers/edit/editOnPaste.js

```diff
@@ -87,9 +96,35 @@ function editOnPaste(editor: DraftEditor, e: SyntheticClipboardEvent): void {
     }
   }
 
-  let textBlocks: Array<string> = [];
   const text = data.getText();
-  const html = data.getHTML();
+  let html = getHTML(data);
+
+  if (text && !html) {
+    // The pasted content has text, but not HTML. For certain browsers (old versions of Safari, IE, and Edge)
+    // the html isn't provided as part of the clipboardData. To work around this, follow the following algorithm:
+    // Do NOT call e.preventDefault(). Instead, we want the browser to paste, just not in the draft element.
```

Comment:
Minor: in this code we are Draft so it seems strange to refer to Draft as a 3rd party